README.md

```diff
@@ -43,14 +43,29 @@ To install necessary dependencies, run the following commands:
 
 3. Create a new database:
    - Open a terminal and run `psql` to enter the PostgreSQL command-line interface.
+   - If you are prompted for a password, you may need to instead run `psql -U postgres -h localhost` and provide the superuser password you provided at set-up.
    - Run the following command to create a new database: `CREATE DATABASE your_database_name;`
 
 4. Create a database user:
    - Run the following command to create a new user: `CREATE USER your_username WITH PASSWORD 'your_password';`
    - Grant privileges to the user for the database: `GRANT ALL PRIVILEGES ON DATABASE your_database_name TO your_username;`
 
 5. Update your application configuration:
-   - In your backend code, update the database connection settings to use the database name, username, and password you created.
+   - In your [backend code](https://github.com/blazickjp/GPT-CodeApp/blob/main/backend/database/my_codebase.py), update the database connection settings to use the database name, username, and password you created.
+
+   OR
+   
+   - Create an environment variable `POSTGRES_AUTH` and supply to a path to a local `.json` you create with the following format:
+      ```json
+      {
+      "dbname": "DB_FROM_STEP3",
+      "user": "USER_FROM_STEP4",
+      "password": "PWD_FROM_STEP4",
+      "host": "localhost"
+      }
+      ```
```

Comment:
Instead of having the env variable point to a json object, let's use a .env file and include `CODEAPP_DB_NAME, CODEAPP_DB_USER, CODEAPP_DB_PW, CODEAPP_DB_HOST`. That way we can include a sample .env.example file with defaults.